src/prettier-printer.d.ts

```diff
@@ -0,0 +1,52 @@
+declare module 'prettier-printer' {
+    interface IDoc {}
```

Comment:
Hmm... I'm not intimately familiar with TypeScript, but given TypeScript's use of structural typing, doesn't this mean that any object is seen as a valid document?  IOW, given this definition, does e.g. `render(0, {foo: 'bar'})` give a type error (it probably should, because it is an error)?